src/widgets/modals/ByProgramModal.js

```diff
@@ -88,8 +89,10 @@ export const ByProgramModal = ({ settings, database, transactionType, onConfirm
   const [state, dispatch] = useReducer(byProgramReducer, {}, () =>
     initialState({ transactionType })
   );
-  const { steps, modalData, program, orderType, period, supplier, storeTags, name } = state;
```

Comment:
Maybe should have taken these tags out of the state in this PR. Hoping to do a bit more refactoring once functionality is in